packages/knip/fixtures/plugins/i18next-parser/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@fixtures/i18next-parser",
+  "version": "*",
+  "scripts": {
+    "i18next": "i18next -c custom.config.js"
```

Comment:
I love that this plugin exports `args`. The value of `-c` indicates there's probably a mismatch between `-c custom.config.js` and the file on disk `i18next-parser.config.js`?